polaris/ocean/tests/inertial_gravity_wave/init.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, test_case, resolution):
                          name=f'init_{resolution}km',
                          subdir=f'{resolution}km/init')
         self.resolution = float(resolution)
+        self.add_output_file(filename='initial_state.nc')
+        self.add_output_file(filename='culled_graph.info')
```

Comment:
```suggestion
        for filename in ['culled_mesh.nc', 'culled_graph.info', 'initial_state.nc']:
            self.add_output_file(filename=filename)
```
We need a third file for the viz step:
https://github.com/altheaden/polaris/blob/fix-inertial-gravity-dependencies/polaris/ocean/tests/inertial_gravity_wave/viz.py#L43
so let's just make it a loop.